clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -128,17 +129,24 @@ class matmul_desc_t {
     trans_a,
     trans_b,
     trans_c,
-    epilogue
+    epilogue,
+    a_scale_pointer,
+    b_scale_pointer,
+    d_scale_pointer,
+    amax_d_pointer,
```

Comment:
how about use full name for amax in  amax_d_pointer? 
```

Reply:
done